packages/ProgressAccordion/src/components/Responses/Responses.js

```diff
@@ -20,7 +22,7 @@ const Responses = ({ children }) => {
   if (!children) return null;
 
   return (
-    <dl data-pka-anchor="responses" css={responsesStyles}>
+    <dl data-pka-anchor="responses" css={responsesStyles} tabIndex="0">
```

Comment:
Just curious why this `dl` need to be focusable by tab?

Reply:
Because it has `max-height`, so it can have a vertical scrollbar... then for keyboard access it needs to be tabbable.